packages/kbn-tinymath/src/functions/round.js

```diff
@@ -22,11 +22,10 @@ const rounder = (a, b) => Math.round(a * Math.pow(10, b)) / Math.pow(10, b);
  * round([2.9234, 5.1234, 3.5234, 4.49234324], 2) // returns [2.92, 5.12, 3.52, 4.49]
  */
 
-module.exports = { round };
-
-function round(a, b = 0) {
+function round(a, b) {
```

Comment:
The refactor here was due to the documentation processor, which handles the default parameter assignment in the function generating a different documentation layout in the `functions.md` file for the `round` signature.

Reply:
Changed the code here with a cleaner solution in a deeper function.